docs/rules/jsx-sort-props.md

```diff
@@ -75,6 +76,24 @@ When `true`, alphabetical order is not enforced:
 <Hello tel={5555555} name="John" />
 ```
 
+### `reservedFirst`
+
+This can be a boolean or an array option.
+
+When `reservedFirst` is defined, React reserved props (`children`, `dangerouslySetInnerHTML`, `key`, `ref`) must be listed before all other props, but still respecting the alphabetical order:
```

Comment:
I'll add the tests for them all shortly :)

Regarding [`dangerouslySetInnerHTML`](https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml), from [playing around with a fiddle](https://jsfiddle.net/69z2wepo/75289/), it seems to have no affect at all on composite components - so it seems to be treated as a "normal" props. I'll add the DOM node check for it.

(Maybe there should be another rule to ensure people don't use `dangerouslySetInnerHTML` on composite components? Seems like something that could easily cause bugs for people who expect it to work.)